cmd/grpc.go

```diff
@@ -25,12 +25,14 @@ func init() {
 	GrpcCMD.Flags().StringVar(&Network, "network", "unix", "network type")
 	GrpcCMD.Flags().StringVar(&AddressFlag, "socket", "@crossplane/fn/default.sock", "set where socket should be located")
 	GrpcCMD.Flags().BoolVar(&DevMode, "devmode", false, "enable dev composition functions")
+	GrpcCMD.Flags().BoolVar(&ExternalConnectionEnabled, "externalconnections", false, "enable support for LoadBalancer to make PostgreSQL accessible from outside the cluster")
 }
 
 var (
-	Network     = "unix"
-	AddressFlag = "@crossplane/fn/default.sock"
-	DevMode     = false
+	Network                   = "unix"
+	AddressFlag               = "@crossplane/fn/default.sock"
+	DevMode                   = false
+	ExternalConnectionEnabled = false
```

Comment:
I wouldn't pass this as a flag.

We usually have these configurations in the composition itself in the config: https://github.com/vshn/component-appcat/blob/master/component/vshn_postgres.jsonnet#L849

They are dynamic and can even be set per composition, this makes them a lot more flexible. You can access the configs via `iof.Config` in the compisition function.